e2e/cypress/integration/default_email.spec.js

```diff
@@ -31,8 +31,9 @@ describe('Distribution and ContactPoints are correct', () => {
             });
 
             //there should be no invalid urls because they would have been removed
+            cy.log(validUrls)
             expect(invalidUrls).to.have.lengthOf(0);
-            expect(validUrls).to.have.lengthOf(16);
+            expect(validUrls).to.have.lengthOf(12);
```

Comment:
![image](https://user-images.githubusercontent.com/74677943/155581813-e39e0ff5-3a57-45d2-b6c0-e4e5d3857ff2.png)
14 distributions with 12 valid urls

Reply:
The invalid URLs will not appear in the datajson file. `package2pod` in the `ckanext-datajson` ckan plugin reads each `accessURL` or `downloadURL` and uses a helper function to validate if it is a browser readable URL. The invalid links will still be stored in the database, but the object containing that link will not be written into the datajson file. 12 is the correct number of valid URLs because the other URLs in the datajson file because the `invalid` urls have not been written.

Reply:
`resource_search` pulls `accessURLs`. The test validates both access and download urls. With 14 access URLs, 4 are invalid in tests leaving 10. The other 2 urls are `downloadURL` which are validated in the test.